xlinectl/src/utils/printer.rs

```diff
@@ -80,6 +168,7 @@ impl SimplePrinter {
 /// Field Printer of common response types
 struct FieldPrinter;
 
+#[allow(dead_code)]
```

Comment:
Please move the this `#[allow(dead_code)]` to the `FieldPrinter::value` method

Reply:
done